Public/Src/Cache/ContentStore/Vsts/DedupReadOnlyContentSession.cs

```diff
@@ -118,13 +128,17 @@ public class DedupReadOnlyContentSession : ContentSessionBase
         /// <param name="implicitPin">Policy determining whether or not content should be automatically pinned on adds or gets.</param>
         /// <param name="dedupStoreHttpClient">Backing DedupStore http client.</param>
         /// <param name="timeToKeepContent">Minimum time-to-live for accessed content.</param>
+        /// <param name="pinInlineThreshold">Maximum time-to-live to inline pin calls.</param>
+        /// <param name="ignorePinThreshold">Minimum time-to-live to ignore pin calls.</param>
         /// <param name="maxConnections">The maximum number of outboud connections to VSTS.</param>
         public DedupReadOnlyContentSession(
             IAbsFileSystem fileSystem,
             string name,
             ImplicitPin implicitPin,
             IDedupStoreHttpClient dedupStoreHttpClient,
             TimeSpan timeToKeepContent,
+            TimeSpan pinInlineThreshold,

```

Comment:
I'm fine so far, but maybe it's time to add a configuration class, something like DedupConfiguration?

Reply:
Right now I don't think there are enough things different in Dedup and Blob constructors, so maybe after some other differences emerge we can reconsider adding something like DedupConfiguration